src/commoncode/codec.py

```diff
@@ -26,13 +26,22 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+
```

Comment:
Repetition of commits https://github.com/nexB/scancode-toolkit/pull/1534/commits/785b48ab0bdf8b18066fe0161f871715d54a9007